README.md

```diff
@@ -26,21 +20,31 @@ In the root directory of the application, create a virtual environment, activate
 pip install -r requirements.txt
 ```
 
-### Download and configure the data file
-
-Download the data file (in `json` format) from the project assignment in Canvas and update the `config.json` with the path to the file. Note, you can also specify an environment variable by the same name as the config setting (`ENPM611_PROJECT_DATA_PATH`) to avoid committing your personal path to the repository.
-
-
 ### Run an analysis
 
-With everything set up, you should be able to run the existing example analysis:
+The file to execute is `run.py`
+Command syntax
 
 ```
-python run.py --feature 0
+python run.py -f | --feature FEATURE [-u | --user USER] [-l | --label LABEL]
 ```
 
-That will output basic information about the issues to the command line.
+### Description of Features
+#### 1. Bug Pattern analysis
+  Analyze and visualize trends in issues based on labels\
+  Options accepted: -u | --user USER (Analyzes for all users if not provided)
```

Comment:
Add sample command for each type of analysis

Reply:
Added command syntax